test/__tests__/CalendarTest.jsx

```diff
@@ -61,11 +86,79 @@ describe('Calendar headeronent test suite', function() {
     let rows = calendar.find('.checkoutCalRow');
     expect(rows.length).toBe(6); // September 2018 has 6 rows in the calendar
     
+    // Click, update props
     rightBtn.simulate('click');
+    calendar.setProps({month, year});
     calendar.update();
-    
+
     rows = calendar.find('.checkoutCalRow');
     expect(rows.length).toBe(5); // October 2018 has 5 rows in the calendar
   });
 
+  it('should display dashes on all cells before today', function() {
+    var thisMonth = new Date().getMonth();
```

Comment:
You might want to switch to const to get more in line with airbnb but not significant

Reply:
Done, used let and also const when possible